### PR TITLE
NamelessMC v2 pre13 and README.md update

### PR DESCRIPTION
**Pull request by PadowYT2**
This is done because of NamelessMC/Documentation#1

**Note: If any updates needed, msg me on discord `PadowYT2#1148`**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/.github/
```
Comment (Aberdeener) [inline .gitignore:1]: The `.github/` folder is used for issue templates + GitHub Actions workflows. It cannot be ignored!

File: README.md [hunk 2]
```diff
@@ -55,21 +41,10 @@ Support can be found in one of the following places:
 - [Official support forum](https://namelessmc.com/forum)
 - [SpigotMC](https://www.spigotmc.org/threads/nameless-minecraft-website-software.34810/)
 
-Feature requests can be posted on the [forum](https://namelessmc.com/forum/view/7-web-feature-requests/), and bugs can be reported in the [GitHub Issues](https://github.com/NamelessMC/Nameless/issues) tab.
```
Comment (Aberdeener) [inline README.md:58]: Before we totally remove this link, we should get the ok from Sam, as I know he would like to keep the website somewhat involved.

File: README.md [hunk 3]
```diff
@@ -81,6 +56,14 @@ We welcome all contributions of code and translations. Please feel free to fork
...
+Just make sure you translate the [latest development version](https://github.com/NamelessMC/Nameless/archive/refs/heads/v2.zip), not the latest release or we won't be able to merge your translations! 
+Currently other available translations can be found [here](https://translate.namelessmc.com). Please note, not all translations may be up to date. To discuss changes with fellow translators, visit the [NamelessMC Translators](https://discord.gg/7Dku3fE) discord server.
+NamelessMC translations are kindly provided by the community. 
+Currently available translations can be found [here](https://github.com/NamelessMC/Nameless/tree/v2/custom/languages). Please note, not all translations may be up to date.
+
 ## Special Thanks
-- All NamelessMC [contributors](https://github.com/NamelessMC/Nameless/graphs/contributors).
-- [JetBrains](https://www.jetbrains.com/), whose products are used to develop the NamelessMC project.
+* All [NamelessMC Translation contributors](https://github.com/NamelessMC/Nameless/CONTRIBUTORS.md).
```
Comment (Aberdeener) [inline README.md:67]: Nitpick: I would swap the order of translation contributors and contributors links

**Review by Aberdeener — CHANGES_REQUESTED** [review]
A few minor changes, but above all, we need to decide if we even want to use your new documentation software.
My vote is to stay with Wiki.js

**Comment by PadowYT2** [comment]
Uh oh, I accidently deleted a rep and not the branch

I've created a new pull request #2549

**Pull request closed without merging**